feerefund.go

```diff
@@ -4,48 +4,48 @@ import (
 	"encoding/json"
 )
 
-// FeeRefundParams is the set of parameters that can be used when refunding a fee.
+// ApplicationFeeRefundParams is the set of parameters that can be used when refunding an application fee.
```

Comment:
@brandur-stripe I realized tonight that I renamed to `ApplicationFeeRefund*` but the resource is really `fee_refund` and called FeeRefund elsewhere. Should I undo this one?

Reply:
I'd just leave it — it's called "application fee refund" in both its name in the reference documentation (i.e., on the left navbar) and in Stripe.NET, and "application fee refund" is the superior name in general. It groups well with "application fee" and describes exactly what it is.

Reply:
Actually, on second thought — I think it kind of sucks how the struct is `ApplicationFeeRefund` but the package is `feerefund`. Given the value of preferring to leave as is where possible, I'd be okay renaming it back to `FeeRefund`.

Reply:
@brandur-stripe fixed, PTAL